added edit mode hints (#976)
tinacms/tinacms.org: content/guides/tina-cloud/starter/connect-to-tina-cloud.md

Make some edits through the sidebar and click save.
Changes are saved in your GitHub repository.

> > Hint: To exit edit mode navigate to the `/exit-admin` route if your running on localhost it is http://localhost:3000/exit-admin .

Reviewer: ```suggestion
> Hint: To exit edit mode, navigate to the `/exit-admin` route.  If you are running on `localhost`, it is `http://localhost:3000/exit-admin`.
```
